homeassistant/components/xiaomi_miio/const.py

```diff
@@ -47,6 +47,8 @@ class SetupException(Exception):
 
 
 # Fan Models
+MODEL_AIRPURIFIER_4 = "zhimi.airp.mb5"
+MODEL_AIRPURIFIER_4PRO = "zhimi.airp.vb4"
```

Comment:
Unless it's called "4Pro" we should separate words with underscore in variable names.
```suggestion
MODEL_AIRPURIFIER_4_PRO = "zhimi.airp.vb4"
```

Reply:
updated naming